### DISCUSSION TITLE
Required Configuration for showing Round Name

### DISCUSSION THREAD

**Discussion by yongshun950824** [Drarig29/brackets-viewer.js#82] [Q&A] [answered]
### Demo & Expected Outcome

I can see the round name _(highlighted with red box)_ in this [Replit host](https://replit.com/@Drarig29/Toornament-and-brackets-viewerjs#public/index.html).

![Screenshot for Replit host](https://user-images.githubusercontent.com/29250581/154630419-48ace8c1-d9e8-4ace-a8a3-8775b91ee9d6.png)

And for the [Repository home page](https://github.com/Drarig29/brackets-viewer.js#brackets-viewerjs) the screenshot was also shown with the Round.

### Step(s) to reproduce

While I clone from this [repository](https://github.com/Drarig29/brackets-viewer.js) and setup in my local environment. Use the exact same file as in the repo, but both Tournament Bracket didn't show the round as from the [first screenshot](https://user-images.githubusercontent.com/29250581/154630419-48ace8c1-d9e8-4ace-a8a3-8775b91ee9d6.png).

Also, I try to attempt for this [Angular demo](https://drarig29.github.io/brackets-docs/) but still the round is not showing.

### Actual Result

Tournament Example

![Tournament Example](https://user-images.githubusercontent.com/29250581/154631091-c767e115-e485-4c9b-9b37-56434add4a16.png)

Tournament Example with Repechage

![Tournament Example with Repechage](https://user-images.githubusercontent.com/29250581/154631116-3cd7de16-c1b1-4e91-8865-d00e5c15afb1.png)

### Question

1. **Request for configuration** for showing the **Round** with customization as [official docs](https://drarig29.github.io/brackets-docs/) wasn't completed.

**Comment by Drarig29** [comment, marked as answer]
For the Angular demo (it was a commission), I needed to hide the headers because it didn't add much information. I hesitated with adding new configuration props to hide headers or using letting the user do it with simple CSS. I chose the latter not to add more clutter in the configuration object.

In the styles for the Angular project, I have this:

https://github.com/Drarig29/brackets-viewer.js/blob/ee61103c80dfea8c294aa69a05e6ee578482a71d/demo/with-printing.html#L15-L28

So I think you should check that you don't have such CSS rule lying around :wink:

**Reply by yongshun950824** [reply]
Oh, thanks for the response. It's worked now. =)

Thanks for the assistance and sorry for didn't realize that CSS part.

![image](https://user-images.githubusercontent.com/29250581/154653207-fa18bb40-ccd0-4e45-a142-725092592eae.png)

**Reply by Drarig29** [reply]
No worries, don't forget to star the projects if you like them ☺
